build.zig

```diff
@@ -0,0 +1,202 @@
+    const microkit_board_dir = b.fmt("{s}/board/{s}/{s}", .{ microkit_sdk_option, @tagName(microkit_board_option), @tagName(microkit_config_option) });
+    std.fs.cwd().access(microkit_board_dir, .{}) catch |err| {
+        switch (err) {
+            error.FileNotFound => std.log.err("Path to '{s}' does not exist\n", .{ microkit_board_dir }),
+            else => std.log.err("Could not acccess path '{s}', error is {any}\n", .{ microkit_board_dir, err })
+        }
+        std.posix.exit(1);
+    };
+
+    // Libmicrokit
+    libmicrokit = LazyPath{ .cwd_relative = b.fmt("{s}/lib/libmicrokit.a", .{ microkit_board_dir }) };
+    libmicrokit_linker_script = LazyPath{ .cwd_relative = b.fmt("{s}/lib/microkit.ld", .{ microkit_board_dir }) };
+    libmicrokit_include = LazyPath{ .cwd_relative = b.fmt("{s}/include", .{ microkit_board_dir }) };
+
+    const printf = b.addObject(.{
+        .name = "printf",
+        .target = target,
+        .optimize = optimize,
+    });
+    printf.addCSourceFiles(.{ .files = &.{ "util/printf.c", "util/putchar_debug.c" } });
+    printf.addIncludePath(b.path("include"));
+    printf.addIncludePath(libmicrokit_include);
+
+    const blk_virt = addPd(b, .{
+        .name = "blk_virt.elf",
+        .target = target,
+        .optimize = optimize,
+        // Microkit expects and requires the symbol table to exist in the ELF,
+        // this means that even when building for release mode, we want to tell
+        // Zig not to strip symbols from the binary.
+        .strip = false,
+    });
+    blk_virt.addCSourceFile(.{ .file = b.path("blk/components/virt.c"), .flags = &.{"-DBLK_NUM_CLIENTS=2"} });
+    blk_virt.addCSourceFiles(.{ .files = &.{ "blk/util/bitarray.c", "blk/util/fsmalloc.c", "blk/util/util.c", "util/cache.c" } });
+    blk_virt.addIncludePath(b.path("include"));
+    blk_virt.addObject(printf);
+    b.installArtifact(blk_virt);
+
+    const serial_virt_rx = addPd(b, .{
+        .name = "serial_virt_rx.elf",
+        .target = target,
+        .optimize = optimize,
+        // Microkit expects and requires the symbol table to exist in the ELF,
+        // this means that even when building for release mode, we want to tell
+        // Zig not to strip symbols from the binary.
+        .strip = false,
+    });
+    serial_virt_rx.addCSourceFile(.{
+        .file = b.path("serial/components/virt_rx.c"),
+        .flags = &.{"-DSERIAL_NUM_CLIENTS=3", "-fno-sanitize=undefined"}
+    });
+    serial_virt_rx.addIncludePath(b.path("include"));
+    serial_virt_rx.addObject(printf);
+    b.installArtifact(serial_virt_rx);
+
+    const serial_virt_tx = addPd(b, .{
+        .name = "serial_virt_tx.elf",
+        .target = target,
+        .optimize = optimize,
+        // Microkit expects and requires the symbol table to exist in the ELF,
+        // this means that even when building for release mode, we want to tell
+        // Zig not to strip symbols from the binary.
```

Comment:
Remove these comments, they were copy-and-pasted but they're not needed anymore.